src/redist/redist.csproj

```diff
@@ -21,6 +21,8 @@
     <PackageReference Include="NuGet.Build.Tasks" Version="$(CLI_NuGet_Version)" />
     <PackageReference Include="Microsoft.TestPlatform.CLI" Version="$(CLI_TestPlatform_Version)" />
     <PackageReference Include="Microsoft.TestPlatform.Build" Version="$(CLI_TestPlatform_Version)" />
+    <PackageReference Include="NuGet.Packaging.Core.Types" Version="4.3.0-preview1-2500" />
```

Comment:
I was wondering about it in the beginning as well. I removed it, and the compiler(CrossGen) will complain about the missing type.

I guess it is due to CrossGen itself cannot resolve this dependency from NuGet packages. When it was in the same repo, NuGet.Packaging was there already.